packages/ia-zendesk-help-widget/README.md

```diff
@@ -0,0 +1,40 @@
+# Zendesk Help Widget Component
+
+## Usage
+
+```html
+<ia-zendesk-help-widget
+  widgetSrc="https://static.zdassets.com/ekr/snippet.js?key="
+  buttonVisible="true"
```

Comment:
If the button is not visible, couldn't you just not add it to the DOM instead of setting a property on it?

Reply:
Good suggestion, lets just not add whole stuff at petabase side.